src/cli/minimist.js

```diff
@@ -1,14 +1,14 @@
-"use strict";
+'use strict';
 
-const minimist = require("minimist");
+const minimist = require('minimist');
```

Comment:
Can you switch these back to double quotes?

Reply:
Yes...  I will ...